picoquic/picosocks.c

```diff
@@ -22,7 +22,7 @@
 #include "picosocks.h"
 #include "picoquic_utils.h"
 
-int picoquic_bind_to_port(SOCKET_TYPE fd, int af, int port)
+int picoquic_bind_to_port(char *ip, SOCKET_TYPE fd, int af, int port)
 {
```

Comment:
You probably do not need that. The picoquic loop uses socket options to specify the outgoing IP address. Binding one socket to a port number is sufficient.